### PR TITLE
add migration for MongoDB supportsProfiler = true

### PR DESCRIPTION
**Pull request by sushi30**
### Describe your changes:

Add migration for MongoDB profiler support.
Depends on:
- https://github.com/open-metadata/OpenMetadata/pull/15189

#
### Type of change:

- [x] Improvement

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [x] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

- [ ] I have added tests around the new logic.
- [ ] For connector/ingestion changes: I updated the documentation.

### CODE REVIEW

**Review by harshach — APPROVED** [review]

**Comment by harshach** [comment]
@sushi30 we are not using the flyway for new migrations. Please use native migrations to add this change

**Review by harshach — CHANGES_REQUESTED** [review]
We need to add this in native migrations

**Comment by sushi30** [comment]
@harshach done

File: bootstrap/sql/migrations/native/1.3.1/mysql/schemaChanges.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+-- Add the supportsProfiler field to the MongoDB connection configuration
+UPDATE dbservice_entity
+SET json = JSON_INSERT(json, '$.connection.config.supportsProfiler', TRUE)
```
Comment (pmbrull) [inline bootstrap/sql/migrations/native/1.3.1/mysql/schemaChanges.sql:3]: should this be 1.3.1 though? If we'll deploy all these NoSQL thingies by 1.3.2 - 1.4.0, might be better to leave the 1.3.1 clean

Comment (sushi30) [reply bootstrap/sql/migrations/native/1.3.1/mysql/schemaChanges.sql]: moved to 1.4.0

**Review by harshach — APPROVED** [review]

**Comment by sushi30** [comment]
resolved conflicts

**Review by ulixius9 — APPROVED** [review]

**Pull request merged**